Makefile

```diff
@@ -179,7 +186,7 @@ clean:
 	uname -a
 	$(CC) --version
 	rm -rf $(BUILD_ROOT)
-	find . \( -name "*loc*.c" -o -name "loc*.h" \)  -exec rm -rf {} \;
+	find ./tests ./test-code \( -name "*loc*.c" -o -name "loc*.h" \)  -exec rm -rf {} \;
```

Comment:
With the addition of support for LOC-ELF scheme, we now have `include/loc.h` and `src/loc.c` part of the checked-in Git files.

So, `clean` cannot clean those up and has to selectively pick-up only the generated `loc.h` and `loc.c` files from test-dirs.